src/main/java/org/broadinstitute/hellbender/tools/spark/sv/AlignAssembledContigsSpark.java

```diff
@@ -22,7 +20,7 @@
 @CommandLineProgramProperties(summary="Align assembled contigs to the reference and call breakpoints from them.",
```

Comment:
My understanding of the code is that this tool no longer calls breakpoint, but delegates work to `CallVariantsFromAlignedContigsSpark`?